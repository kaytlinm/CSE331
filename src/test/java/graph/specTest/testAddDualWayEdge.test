#Add Dual Way Edge Test File
#Tests that a graph can be created, two edges can be created, and two edges can be added in opposing directions.

CreateGraph graph1
AddNode graph1 node1
AddNode graph1 node2
AddEdge graph1 node1 node2 edge1
AddEdge graph1 node2 node1 edge2