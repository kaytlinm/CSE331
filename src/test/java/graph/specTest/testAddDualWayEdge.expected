#Add Dual Way Edge Test File
#Tests that a graph can be created, two edges can be created, and two edges can be added in opposing directions.

created graph graph1
added node node1 to graph1
added node node2 to graph1
added edge edge1 from node1 to node2 in graph1
added edge edge2 from node2 to node1 in graph1
